src/main/jbake/content/using-git.ad

```diff
@@ -0,0 +1,113 @@
+must remember to check if all tests pass, as well as other quality checks such as code style.
+
+## Cloning the OpenNLP Git repository
+
+After obtaining committership to OpenNLP, you probably want to submit your changes to the project source repository.
+This section contains the steps that every committer must follow, in order to make sure every developer is following
+the workflow, and have a consistent and simple commit tree.
+
+    git clone https://git-wip-us.apache.org/repos/asf/opennlp.git
+    cd opennlp
+    git config user.name "Your Name"
+    git config user.email "your-email"
+    git config merge.ff only
+
+You can also clone the project web site repository.
+
+    https://git-wip-us.apache.org/repos/asf/opennlp.git
+    # repeat remaining steps as above
+
+In order to test your commit rights, and following a project tradition, normally the first commit of every new
+member is to add his/her name to the list of project members. Look for the `team.ad` source file in the web site
+repository, add your name, and try sending your first commit.
+
+For a complete list of the project repositories, visit the link:/source-code.html[Source Code] section.
+
+## Merging Pull Requests
+
+This section documents the process of merging code changes contributed via
+link:https://help.github.com/articles/about-pull-requests/[Github Pull Requests]. It is important to
+remember to **always merge with link:https://git-scm.com/docs/git-merge[fast-forward]**. If you followed the steps in
+the first section, your local working copy should be already configured for that. Otherwise, remember to use `ff-only`
+when merging.
+
+### Adding a remote repository pointing to GitHub
+
+In order to fetch the pull requests in GitHub, you need to add a remote repository.
+
+    git remote add github https://github.com/apache/opennlp.git
+    git fetch --all
+    git fetch github pull/<PULL-ID>/head:<BRANCHNAME>
+    git checkout <BRANCHNAME>
+
+Replacing `<PULL-ID>` by the GitHub pull request ID (you can find it in the pull request URL) and `<BRANCHNAME>` by
+the name of the new local branch. If you have suggestions to enhance or fix the pull request, send your comments via
+the GitHub user interface, or add a comment to the JIRA ticket &mdash; if any.
+
+Once you are happy with the changes, you can check out the master branch, and merge the pull request. Remember
+to make sure the **branch has been rebase'd against master**, and also that all tests pass.
+
+    git checkout master
+    git merge --ff-only <BRANCHNAME>
+    git push origin master
+
+In case other commits happened after the pull request was submitted, you must ask the user to rebase the pull
+request against the master branch, and squash his commit. An alternative for that, is to rebase and squash the commits
+yourself, when there is no feedback from the user.
+
+    git checkout <BRANCHNAME>
+    git rebase master
+    # squash and amend the commit message if necessary...
```

Comment:
We only accept PRs that have a single commit, then we don't have to deal with that during merging.